### PR TITLE
Feature/UI stylings

### PR DESCRIPTION
**Pull request by DavidCramer**

### CODE REVIEW

File: ui-definitions/settings-sidebar.php [hunk 1]
```diff
@@ -53,11 +59,18 @@
 	array(
 		'type'        => 'panel',
 		'title'       => __( 'Optimization level', 'cloudinary' ),
-		'description' => __( '40% Optimized', 'cloudinary' ),
+		'description' => function () {
+			$instance = get_plugin_instance()->settings->get_setting( 'sidebar.panel_1.opt_level_0' );
+
+			/* translators: %s is the percentage optimized. */
+
+			return sprintf( '%s Optimized', $instance->get_component()->calculate_percentage() . '%' );
```
Comment (pereirinha) [inline ui-definitions/settings-sidebar.php:67]: @DavidCramer 

This is not a translatable string. :)

**Review by pereirinha — COMMENTED** [review]
@DavidCramer 

Left you a few comments.

File: php/class-settings.php [hunk 2]
```diff
@@ -217,8 +217,8 @@ protected function register_storage( $slug ) {
 	 *
 	 * @return string
 	 */
-	protected function get_storage_key( $slug, $type = null ) {
-		if ( null === $type ) {
+	public function get_storage_key( $slug, $type = null ) {
+		if ( null == $type ) {
```
[the remark this answers is not in the document]
Comment (DavidCramer) [reply php/class-settings.php:221]: This is what happens when you restore a attach and don't check 🤦‍♂️

File: php/delivery/class-responsive-breakpoints.php [hunk 3]
```diff
@@ -136,7 +136,7 @@ protected function create_settings() {
 	 */
 	public function register_settings( $pages ) {
 
-		$pages['responsive']['settings'][0][1][0] = array(
+		$pages['responsive']['settings'][0][2][0] = array(
```
[the remark this answers is not in the document]
Comment (DavidCramer) [reply php/delivery/class-responsive-breakpoints.php:139]: These are purely temporary, since V3 moved then into the true area.

**Pull request merged**